packages/web-app-files/src/views/LocationPicker.vue

```diff
@@ -486,7 +486,6 @@ export default {
   height: auto;
   z-index: 1;
   grid-area: header;
-  background-color: var(--oc-color-background-default);
```

Comment:
Very weird behaviour when moving a file and scrolling (given enough files in the target folder). Row background colors are different for files&folders, also glitchy in the AppBar (which now is transparent)

Reply:
Files are disabled as a target in copy/move scenarios, that's why they have different colors in the LocationPicker view. Fixed the transparent background of the AppBar.